api/envoy/config/listener/v3/listener.proto

```diff
@@ -275,4 +280,13 @@ message Listener {
   // :ref:`use_original_dst <envoy_api_field_config.listener.v3.Listener.use_original_dst>`
   // to true. Default is true.
   google.protobuf.BoolValue bind_to_port = 26;
+
+  // Used to represent an internal listener which does not listen on OSI L4 address but can be used by the
+  // :ref:`envoy cluster <envoy_v3_api_msg_config.cluster.v3.Cluster>` to create a user space connection to.
+  // The internal listener acts as a tcp listener. It supports listener filters and network filter chains.
+  // There are some limitations are derived from the implementation. The known limitations include `connection_balance_config`
```

Comment:
Can you enumerate these as bullet points in full detail?

Reply:
added